frontend/src/app/modules/challenges/challenges.module.ts

```diff
@@ -1,21 +1,51 @@
+import { LayoutModule } from '@angular/cdk/layout';
 import { NgModule } from '@angular/core';
+import { NgbModule, NgbNavModule } from '@ng-bootstrap/ng-bootstrap';
 import { SharedModule } from '@shared/shared.module';
+import { AngularSplitModule } from 'angular-split';
+import { MarkdownModule } from 'ngx-markdown';
+import { MonacoEditorModule, NgxMonacoEditorConfig } from 'ngx-monaco-editor-v2';
 
 import { ChallengesDropdownSelectComponent } from './challenges-dropdown-select/challenges-dropdown-select.component';
 import { ChallengesTestPageComponent } from './challenges-test-page/challenges-test-page.component';
 import { EditorQuestionComponent } from './editor-question/editor-question.component';
+import { OnlineEditorPageComponent } from './online-editor/online-editor-page/online-editor-page.component';
 import { SolutionPageComponent } from './solution-page/solution-page.component';
 import { StepsOfProgressComponent } from './steps-of-progress/steps-of-progress.component';
 import { ChallengesRoutingModule } from './challenges-routing.module';
 
+const monacoConfig: NgxMonacoEditorConfig = {
```

Comment:
I think we don't need it as we already have monaco editor added to shared module

Reply:
![image](https://github.com/BinaryStudioAcademy/bsa-2023-leetwars/assets/68563967/42c2e48e-81d3-4bf2-8166-2fd1e9db13e5)

monacoConfig is not configured in the shared module

i think it should stay here😶

Reply:
but i'm not sure

Reply:
Yeah, I think code-editor must be updated because of it. Also, I think it's up to you to decide to change it or not, because you didn't create online-editor page